src/main/groovy/edu/oregonstate/mist/osuevents/resources/FeedResource.groovy

```diff
@@ -58,11 +59,15 @@ class FeedResource extends Resource {
      */
     @GET
     @Timed
-    Response getFeed() {
+    Response getFeed(@QueryParam("changedInPastHours") Integer changedInPastHours) {
         CsvMapper mapper = new CsvMapper()
         CsvSchema schema = mapper.schemaFor(FeedEvent.class).withHeader()
 
-        List<Event> events = eventsDAOWrapper.getEvents()
+	   if (changedInPastHours <= 0) {
```

Comment:
If I don't provide this parameter, it returns a bad request. I think we should check for negative only if the parameter is non-null.